### DISCUSSION TITLE
Copy an activity

### DISCUSSION THREAD

**Discussion by irwinwilliams** [microsoft/BotFramework-Composer#5648] [General]
![image](https://user-images.githubusercontent.com/1428703/106192608-cdb89180-6182-11eb-8973-33119fb21d4e.png)
It would be cool if you could click the vertical ellipsis and get an option to copy an activity.

**Comment by a-b-r-o-w-n** [comment]
@irwinwilliams did you know you are able to copy from the edit dropdown menu? 

![image](https://user-images.githubusercontent.com/3619060/106202274-84f6dd80-616e-11eb-803a-21514a8593b1.png)

This is still a valid request, but this was an explicit design decision due to the fact that you are able to copy multiple actions at the same time. If copy is exposed on each action node, it may be confusing what the expected copy behavior should be (copy just this node, or all nodes that are selected).
